docs/sources/tempo/getting-started/docker-example.md

```diff
@@ -3,8 +3,26 @@ title: Quick start for Tempo
 menuTitle: Quick start for Tempo
 description: Use Docker to quickly view traces using K-6 and Tempo
 weight: 600
+killercoda:
+  title: Quick start for Tempo
+  description: Use Docker to quickly view traces using K-6 and Tempo
+  details:
+      intro:
+         foreground: docker-compose-update.sh
+  preprocessing:
+    substitutions:
+      - regexp: ../../
+        replacement: /docs/tempo/latest/
+      - regexp: '{{< relref "./instrumentation" >}}'
```

Comment:
@jdbaldry I know the Witers' Toolkit recommends using full URLs. Should I update the URls to all use the full URLs or does it matter that much?

Reply:
Using full URLs would let us remove these preprocessing statements which would be nice. The transformer tool knows how to convert full URLs with version substitution into Killercoda URLs.

Reply:
Okay. If I change the URLs in the body, then the regexp can be removed?

Reply:
Okay. I'll remove the preprocess notes from the frontmatter.